### ISSUE TITLE
Parametric curve: clips when setting control point to min/max

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#579]
Originally reported on Google Code with ID 590

```
Exposure tab, Tone curve - Parametric.
When moving the left control point right under the curve to 0 (extreme left) the image
clips. No clipping at 0,01. Same when setting the right control point to 1, no clipping
at 0,99. 

Branch_3.0, 3a2.14, Ubuntu 10.10/64. 
```

Reported by `paul.matthijsse4` on 2011-03-21 14:49:23

**Comment by paulmatthijsse** [comment]
Hello, I noticed again a clipping problem with the parametric curve. Set the Shadows slider to +1, drag the right control point under the curve to the right to position +1.00, the middle control point to the left and the curve clips, so the image. Clipping starts with the right point at 0.93. See attachment. 

![rt_param_curve_clips](https://user-images.githubusercontent.com/14089212/112807117-d391fc80-9077-11eb-9e6b-e07813dc971c.jpg)

Unrelated: the naming of the sliders Shadows (first) and Darks (second) is a bit confusing, to me darks are darker than shadows! In the Dutch translation of ART I changed this to Shadows for the upper slider and Darker parts for the second slider. Perhaps a suggestion for RT as well?

Version: 5.8-2831-ga64660bfa
Branch: dev

Regards, Paul.
